.github/renovate.json

```diff
@@ -19,36 +39,74 @@
         "^@types/eslint",
         "^@typescript-eslint/",
         "^eslint-",
-        "^eslint$"
+        "eslint$"
       ]
     },
+    {
+      "enabled": true,
+      "groupName": "GraphQL",
+      "matchManagers": ["npm"],
+      "matchPackagePatterns": ["^@graphql/", "^graphql-", "graphql$"]
+    },
     {
       "enabled": true,
       "groupName": "Jest",
       "matchManagers": ["npm"],
-      "matchPackagePatterns": ["^@types/jest$", "^jest", "^ts-jest$", "^jest-"]
+      "matchPackagePatterns": ["^jest-", "jest$"]
+    },
+    {
+      "enabled": true,
+      "groupName": "OpenTelemetry",
+      "matchManagers": ["npm"],
+      "matchPackagePatterns": ["^@opentelemetry/"]
     },
     {
       "enabled": true,
       "groupName": "Playwright",
       "matchManagers": ["npm"],
-      "matchPackagePatterns": ["^@playwright/", "^playwright$", "^playwright-"]
+      "matchPackagePatterns": ["^@playwright/", "^playwright-", "playwright$"]
+    },
+    {
+      "enabled": true,
+      "groupName": "Prettier",
+      "matchManagers": ["npm"],
+      "matchPackagePatterns": ["^prettier", "prettier$"]
     },
     {
       "enabled": true,
       "groupName": "ProseMirror",
       "matchManagers": ["npm"],
-      "matchPackagePatterns": ["^prosemirror-"]
+      "matchPackagePatterns": ["^prosemirror", "prosemirror$"]
     },
     {
       "enabled": true,
       "groupName": "Sentry",
       "matchManagers": ["npm"],
       "matchPackagePatterns": ["^@sentry/"]
     },
+    {
+      "enabled": true,
+      "groupName": "Signia",
+      "matchManagers": ["npm"],
+      "matchPackagePatterns": ["^@signia/"]
+    },
+    {
+      "matchManagers": ["poetry"],
+      "commitMessageTopic": "Python package `{{depName}}`",
+      "additionalBranchPrefix": "py/",
+      "reviewers": ["team:Python"]
+    },
+    {
+      "enabled": true,
+      "groupName": "Pydantic",
+      "matchManagers": ["poetry"],
+      "matchPackagePatterns": ["^pydantic"]
+    },
     {
       "matchManagers": ["cargo"],
-      "commitMessageTopic": "Rust crate `{{depName}}`"
+      "commitMessageTopic": "Rust crate `{{depName}}`",
+      "additionalBranchPrefix": "rs/",
+      "reviewers": ["team:Rust"]
```

Comment:
Do we want to add this to the toolchain groups as well?

Reply:
(The branch prefix and the reviewers)

Reply:
@TimDiekmann I think this is done in https://github.com/hashintel/hash/commit/b8d865a7105bd4cb721672d9d6ad3b2321a6a4af -- if you could check I've caught all Rust crates as well as the toolchain updates which use the custom regex manager approach that'd be great.

Reply:
Looks good, let's try it 🙂